bin/install

```diff
@@ -237,8 +237,17 @@ download_and_verify_checksums() {
     fi
 
     (
-      if [ -z "${GNUPGHOME:-}" ] && [ -d "${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs" ]; then
-        export GNUPGHOME="${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs"
+      if [ -z "${GNUPGHOME:-}" ]; then
+        if [ -n "${ASDF_NODEJS_KEYRING_DIR:-}" ]; then
+          export GNUPGHOME="$ASDF_NODEJS_KEYRING_DIR"
+        else
+          for dir in "${ASDF_DATA_DIR:-$HOME/.asdf}" "${ASDF_DIR:-$HOME/.asdf}" ;do
+            if [ -d "$dir/keyrings/nodejs" ]; then
+              export GNUPGHOME="$dir/keyrings/nodejs"
+              break
+            fi
+          done
+        fi
```

Comment:
I think we can simplify this code to be a single loop, we can also use bash brace expansions to simplify some code, how about:
```suggestion
      for dir in "$ASDF_NODEJS_KEYRING_DIR" "$GNUPGHOME" {"$ASDF_DATA_DIR","$ASDF_DIR","$HOME/.asdf}"/keyrings/nodejs"; do
        if [ -d "$dir" ]; then
          export GNUPGHOME="$dir"
          break
        fi
      done
```